test/fixtures/manifest.json

```diff
@@ -1,14 +1,33 @@
 {
```

Comment:
Yes. I mainly left it in as an example for anyone digging in the tests, but it's also nice for preventing a regression and to represent another common filetype in the compression/manifest creation process. I could see addition of other common filetypes in the future, such as fonts. It really just depends on the desired testing strategy.